vuecomponents/bundle/src/main/bundle/dynamic-rubric_eu.properties

```diff
@@ -0,0 +1,8 @@
+criterion_default=Texto por defecto para el criterio
+edit_criterions=Editar criterios
+confirm_changes=Confirmar cambios en criterios
+cancel_changes=Cancelar cambios en criterios
+add_criterion=Añadir nuevo criterio
+confirm_recalculation=¿Estás seguro de querer aplicar estos cambios a la rúbrica? \
+Modificar los criterios y sus puntuaciones ejecutará un recálculo de las notas de otros estudiantes que tengan seleccionados uno o más criterios afectados por este cambio.  \
+Tras ese proceso la pantalla se recargará y se podrá ver la lista actualizada de puntuaciones para esta pregunta.
```

Comment:
Hi @PabloSanRoman, could you help with these translations please?

Reply:
Hi @bgarciaentornos !

+ criterion_default=Lehenetsitako testua irizpidearentzat
+ edit_criterions=Editatu irizpideak
+ confirm_changes=Berretsi aldaketak
+ cancel_changes=Utzi aldaketak
+ add_criterion=Gehitu irizpidea
+ confirm_recalculation= Ziur zaude gehitu nahi dituzula aldaketa hauek?  \
Irizpideak eta puntuazioak aldatzeak ikasleen notak berriro kalkultzea eragingo du.  \
Prozesu horren ondoren pantaila berriz kargatuko da eta puntuazio eguneratuak ikusi ahal izango dira.